client/ayon_core/plugins/publish/integrate_review.py

```diff
@@ -0,0 +1,98 @@
+import os
+
+import pyblish.api
+import ayon_api
+from ayon_api.server_api import RequestTypes
+
+from ayon_core.lib import get_media_mime_type
+from ayon_core.pipeline.publish import get_publish_repre_path
+
+
+class IntegrateAYONReview(pyblish.api.InstancePlugin):
+    label = "Integrate AYON Review"
+    # Must happen after IntegrateNew
+    order = pyblish.api.IntegratorOrder + 0.15
+
+    def process(self, instance):
+        project_name = instance.context.data["projectName"]
+        src_version_entity = instance.data.get("versionEntity")
+        src_hero_version_entity = instance.data.get("heroVersionEntity")
+        for version_entity in (
+            src_version_entity,
+            src_hero_version_entity,
+        ):
+            if not version_entity:
+                continue
+
+            version_id = version_entity["id"]
+            self._upload_reviewable(project_name, version_id, instance)
+
+    def _upload_reviewable(self, project_name, version_id, instance):
+        ayon_con = ayon_api.get_server_api_connection()
+        major, minor, _, _, _ = ayon_con.get_server_version_tuple()
+        if (major, minor) < (1, 3):
+            self.log.info(
+                "Skipping reviewable upload, supported from server 1.3.x."
+                f" User server version {ayon_con.get_server_version()}"
+            )
+            return
+
+        uploaded_labels = set()
+        for repre in instance.data["representations"]:
+            repre_tags = repre.get("tags") or []
+            # Ignore representations that are not reviewable
+            if "webreview" not in repre_tags:
+                continue
+
+            # exclude representations with are going to be published on farm
+            if "publish_on_farm" in repre_tags:
+                continue
+
+            # Skip thumbnails
+            if repre.get("thumbnail") or "thumbnail" in repre_tags:
+                continue
+
+            # include only thumbnail representations
+            repre_path = get_publish_repre_path(
+                instance, repre, False
+            )
+            if not repre_path or not os.path.exists(repre_path):
+                # TODO log skipper path
+                continue
+
+            content_type = get_media_mime_type(repre_path)
+            if not content_type:
+                self.log.warning("Could not determine Content-Type")
```

Comment:
Would it be worth logging the `repre_path` here as well so the user is aware of what was checked?
```suggestion
                self.log.warning(
                    "Could not determine Content-Type for: %s", repre_path)
```